src/Data/Fin/Base.agda

```diff
@@ -131,6 +131,10 @@ splitAt zero    i       = inj₂ i
 splitAt (suc m) zero    = inj₁ zero
 splitAt (suc m) (suc i) = Sum.map suc id (splitAt m i)
 
+-- inverse of above function
+join-⊎ : ∀ m n → Fin m ⊎ Fin n → Fin (m ℕ.+ n)
```

Comment:
Any objections to calling this simply `join`?

Reply:
No objection at all. I was just worried with name clashes with other uses of 'join' in the library. I'll rename.